client/src/register.css

```diff
@@ -1,13 +1,3 @@
 body {
```

Comment:
This file has been deleted so I would recommend creating main.css where this can be specified along with other general settings like fonts, css variables, and so on.

Reply:
I have created main.css and placed the body in there. Thank you.